### PR TITLE
Feature/slot track processing 131

### PR DESCRIPTION
**Pull request by guiguem**

### CODE REVIEW

File: Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+                    sumP[iTimeBin] += pIt->fAmplitude;
+                    mean[iTimeBin] += pIt->fMean;
+                    variance[iTimeBin] += pIt->fVariance;
+                    neighborhoodAmplitude[iTimeBin] += pIt->fNeighborhoodAmplitude;
+
+                    trackSNR[iTimeBin] += pIt->fAmplitude/pIt->fMean;
+                    trackNUP[iTimeBin] += (pIt->fAmplitude - pIt->fMean) / pIt->fVariance;
+
+                    wideTrackSNR[iTimeBin] += pIt->fNeighborhoodAmplitude/pIt->fMean;
+                    wideTrackNUP[iTimeBin] += (pIt->fNeighborhoodAmplitude - pIt->fMean) / pIt->fVariance;
```
Comment (cclaessens) [inline Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc]: NUP += residual / sqrt(pIt->fVariance)?

File: Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        if (node == NULL) return false;
+
+        SetSlopeMinimum(node->get_value("min-slope", GetSlopeMinimum()));
+        SetProcTrackMinPoints(node->get_value("min-points", GetProcTrackMinPoints()));
+        SetProcTrackAssignedError(node->get_value("assigned-error", GetProcTrackAssignedError()));
+
+        return true;
+    }
+
+    template<typename TracklikeCandidate>
```
Comment (cclaessens) [inline Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc]: this works? For some reason I thought template methods needed to be in the header file.

Comment (guiguem) [reply Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc]: I thought that too, but it seems to work there too. My apologies if I impose an useless standard on you. I don't know if there is a standard already in place for these

File: Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc (hunk 1 above)
Comment (guiguem) [reply Source/EventAnalysis/KTTrackProcessingWeightedSlope.cc]: You are right, but I don't think I did this piece though. I can modify this and solve it before merging

File: Source/Executables/Validation/TestTrackProcessing.cc [hunk 3]
```diff
@@ -10,7 +10,7 @@ and tests the behavior of the algorithms in this processor.
 
 #include "KTLogger.hh"
 
-#include "KTTrackProcessing.hh"
+#include "KTTrackProcessingWeightedSlope.hh"
 #include "KTProcessedTrackData.hh"
 #include "KTSparseWaterfallCandidateData.hh"
```
Comment (cclaessens) [inline Source/Executables/Validation/TestTrackProcessing.cc]: Do you want to do the test for KTSequentialLineData too?

**Pull request merged**